### ISSUE TITLE
Ctrl++ for zoom in does not work on Ubuntu 21.10

### ISSUE THREAD

**Issue by emcangi** [BoostIO/BoostNote.next-local#57]
# Current behavior

The shortcut Ctrl++ does not work, but Ctrl+- does (for zoom out). Zooming in can only be accomplished via the View menu. 

# Expected behavior

The shortcut should work :)

# Steps to reproduce

1. Press Ctrl++ to zoom in

# Environment

- Boost Note.next version: 0.22.3
- OS version and name: Ubuntu 21.10

---

Thank you for your work maintaining Boostnote!

**Comment by Komediruzecki** [comment]
Known issue, due to different keyboard setup, the shortcut is actually `ctrl+shift++`, can you try with that and see if it works for you?

You can check which layout and how application deals with key events at https://keycode.info

**Comment by emcangi** [comment]
Yes, that works. This is a different keyboard setup in Linux or something related to my specific keyboard?

**Comment by Komediruzecki** [comment]
> Yes, that works. This is a different keyboard setup in Linux or something related to my specific keyboard?

Related to specific keyboard layout used, and how Web browser interprets it.
